### PR TITLE
Add tutorial for Ctrl C Signal Bot

### PR DESCRIPTION
**Pull request by sbis04**
I have added the tutorial for the **Ctrl C Signal Bot** under the `Tutorials > Examples > Professional` section. Also, updated the link on the tutorials overview page of this bot to the correct one.

The page looks like this:

CleanShot 2021-08-12 at 17 24 36@2x

Tag #891

### CODE REVIEW

File: docusaurus/docs/examples/professional/ctrl-c-signal-bot.mdx [hunk 1]
```diff
@@ -0,0 +1,455 @@
...
+  }
+  console.log(qrcodeValueToImageUrl(qrcode))
+  console.log('^^^ Online QR Code Image URL ^^^ ')
+  console.log(`[${status}] ${qrcode} Scan QR Code above url to log in: `)
+}
+```
+
+**onMessage**
+
+This will retrieve the message send by someone to your account and print it on console.
```
Comment (Rohitesh-Kumar-Jain) [inline docusaurus/docs/examples/professional/ctrl-c-signal-bot.mdx]: send -> sent ?
on console -> on the console?

**Review by Rohitesh-Kumar-Jain — APPROVED** [review]
Thank you for wonderful documents : )

**Review by proudofsimin — APPROVED** [review]

File: docusaurus/docs/examples/professional/ctrl-c-signal-bot.mdx (hunk 1 above)
Comment (sbis04) [reply docusaurus/docs/examples/professional/ctrl-c-signal-bot.mdx]: Have corrected it now.

**Review by shwetalsoni — APPROVED** [review]

**Comment by sbis04** [comment]
@Rohitesh-Kumar-Jain @proudofsimin I think this PR can be added to #1056

**Review by lijiarui — APPROVED** [review]
Good job!

**Pull request merged**